apps/judicial-system/backend/src/app/modules/notification/notification.service.ts

```diff
@@ -853,6 +869,8 @@ export class NotificationService {
           }),
       defenderName ?? '',
       defenderEmail ?? '',
+      [],
```

Comment:
Is there a reason this Attachment argument is a empty array while the others are undefined ?

Reply:
Nope, just a mistake..